packages/ketcher-core/src/domain/entities/atom.ts

```diff
@@ -21,6 +21,8 @@ import { Elements } from 'domain/constants';
 import { Pile } from './pile';
 import { Struct } from './struct';
 
```

Comment:
Excellent job! I am just curious that is there any difference between getPickUnion and the built-in Pick?